noxfile.py

```diff
@@ -0,0 +1,25 @@
+import nox
```

Comment:
Could you include a docstring of a sentence length saying a noxfile is like a makefile, and use of it is documented in the readme you linked to? I think such comments are super-helpful overall when onboarding myself in new projects.

Reply:
added two suggestions, what do you think?

Reply:
Super!